Problem with process.exit

```bash
> scratch@1.0.0 test
> jest

PASS dist/tests/configurations.test.js
PASS src/tests/configurations.test.ts
PASS dist/tests/exceptions.test.js
PASS dist/tests/routes.test.js
PASS src/tests/routes.test.ts
PASS src/tests/exceptions.test.ts

Test Suites: 6 passed, 6 total
Tests:       10 passed, 10 total
Snapshots:   0 total
Time:        12.056 s
Ran all test suites.
UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Error: 404: Not Found".
Error: Process completed with exit code 1.
```

--- a/README.md
+++ b/README.md
@@ -1,5 +1,5 @@
 # Scout Backend
-![APM](https://img.shields.io/apm/l/vim-mode) ![Statements](https://img.shields.io/badge/statements-97.22%25-brightgreen.svg?style=flat) ![Branches](https://img.shields.io/badge/branches-89.65%25-yellow.svg?style=flat) ![Functions](https://img.shields.io/badge/functions-90.9%25-brightgreen.svg?style=flat) ![Lines](https://img.shields.io/badge/lines-97.12%25-brightgreen.svg?style=flat)
+![APM](https://img.shields.io/apm/l/vim-mode) ![Statements](https://img.shields.io/badge/statements-97.93%25-brightgreen.svg?style=flat) ![Branches](https://img.shields.io/badge/branches-89.65%25-yellow.svg?style=flat) ![Functions](https://img.shields.io/badge/functions-100%25-brightgreen.svg?style=flat) ![Lines](https://img.shields.io/badge/lines-97.85%25-brightgreen.svg?style=flat)
 ## .env
 ```ini
 #app

--- a/src/handlers/errors/CriticalErrorHandler.ts
+++ b/src/handlers/errors/CriticalErrorHandler.ts
@@ -6,10 +6,10 @@ import { TelegramLogger } from '../../loggers/vendor/TelegramLogger';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const CriticalErrorHandler: ErrorRequestHandler = (err: Error | BaseError, req: Request, _res: Response, _next: NextFunction): void => {
-  Promise.all([TelegramLogger.log(err, req), ErrorLogger.log(err, req)]).then(() => {
-    if (!(err instanceof BaseError)) {
-      DebugLogger.debug('Critical error. Emergency application shutdown');
-      process.exit(1);
-    }
-  });
+  TelegramLogger.log(err, req);
+  ErrorLogger.log(err, req);
+  if (!(err instanceof BaseError)) {
+    DebugLogger.debug('Critical error. Emergency application shutdown');
+    process.exit(1);
+  }
 };

